TradeItIosTicketSDK2/TradeItLinkedBroker.swift

```diff
@@ -94,7 +94,9 @@ import PromiseKit
     public func refreshAccountBalances(onFinished: @escaping () -> Void) {
         let promises = accounts.map { account in
             return Promise<Void> { fulfill, reject in
-                account.getAccountOverview(onSuccess: fulfill, onFailure: { errorResult in
+                account.getAccountOverview(onSuccess: { _ in
```

Comment:
You asked for `getAccountOverview` to pass in the account overview... you have to handle the new parameter. `fulfill` takes no parameters, hence you can't just set `onSuccess: fulfill`.